frontend/src/config/jordan/prism.json

```diff
@@ -1,6 +1,6 @@
 {
   "country": "Jordan",
-  "alertFormActive": false,
+  "alertFormActive": true,
```

Comment:
Was this meant to be committed?

Reply:
Yes, temporarily for the auto deploy to show the alerts